src/content/development/release-process/_index.en.md

```diff
@@ -99,7 +100,10 @@ To do so, run the `make` command with `dryrun=true`.
 {{% /notice %}}
 
 The command runs, gathers the data for the release, updates the release YAML and pushes it for review. Once the review process is done,
-and the automated release steps on the CI have finished successfully, simply **run the same command again to advance to the next stage**.
+and the automated release steps on the CI have finished successfully, merge the PR. Pull requests will then be created for all dependent
```

Comment:
```suggestion
, merge the PR. Pull requests will then be created for all dependent
```

The original meant the automation that runs after the PR is merged, your change makes it about the tests on the PR. I think the result is a bit redundant, we don't need to explain that tests have to pass as a part of the review process and you talk about the post-merge automation later.